src/coreComponents/dataRepository/GroupContext.cpp

```diff
@@ -38,21 +38,9 @@ string GroupContext::toString() const
   bool foundNearestLine = false;
   for( Group const * parentGroup = &m_group; parentGroup->hasParent(); parentGroup = &parentGroup->getParent() )
   {
-    if( !foundNearestLine && parentGroup->getDataContext().isDataFileContext() )
+    if( !foundNearestLine )
```

Comment:
Would we have something like
```c++
string const path = foundNearestLine ? parentGroup->getName() : parentGroup->getDataContext().getTargetNameInPath( foundNearestLine );
path.insert( 0, '/' + path );
```
make sense?
Would it make sense, based on the value of `foundNearestLine` to make `getTargetNameInPath` return either `parentGroup->getName()` or its actual content?
I do not have the big picture so I do not know... Tell me what you think.

I'm surely mistaken, but it looks to me that `foundNearestLine` is intended for optimization mostly?

Reply:
`foundNearestLine` is used here to only insert the line of the nearest `DataFileContext`. The goal is to not output the line of the further Groups because, according to our observations, they does not help the user.
I previously used `isDataFileContext()` for the same purpose, because I need a way to determine if a file line has already been found during the path construction.